Saule/LinkType.cs

```diff
@@ -23,6 +23,11 @@ public enum LinkType
         /// </summary>
         Related = 2,
 
+        /// <summary>
+        /// Only top section links
+        /// </summary>
+        Top = 4,
```

Comment:
Like I said on the issue, should this contain the word `Self` since it only applies to that?

Reply:
Updated it. Now it's `TopSelf`